bin/oneoff/data_fix_for_missing_emails

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+#
+# This script reads a CSV of teacher email addresses. For each, it  checks to
+# see if there is a user account without a plaintext email matching the email
+# address (comparing the hashes). If there is, it upgrades the (student) account
+# to a teacher account, populating the plaintext email field with the email
+# address.
+#
+# This is being done to alleviate account issues impacting the PLC team.
+#
+# Note that this script assumes a list of email addresses within the file
+# /tmp/teacher_emails.csv. Since this file contains PII, it will live only
+# temporarily (also not within source control) at that location.
+
+require 'csv'
+require_relative '../../dashboard/config/environment'
+
+CSV.foreach('/tmp/teacher_emails.csv') do |row|
+  email = row[0]
```

Comment:
Not that important but it looks like this isn't really a csv, but rather list of emails, one per line. You could do
```ruby
File.open('/tmp/teacher_emails') do |line|
  email = line.strip
...
```